pkg/pillar/go.mod

```diff
@@ -12,9 +12,7 @@ require (
 	github.com/containerd/typeurl v1.0.2
 	github.com/cshari-zededa/eve-tpm2-tools v0.0.4
 	github.com/digitalocean/go-qemu v0.0.0-20181112162955-dd7bb9c771b8
-	github.com/docker/cli v20.10.13+incompatible // indirect
-	github.com/docker/distribution v2.8.1+incompatible // indirect
-	github.com/docker/docker v20.10.13+incompatible
+	github.com/docker/docker v20.10.17+incompatible
```

Comment:
Any particular reason for updating these vendored packages as part of this PR?
If we need to update vendored packages it presumably makes sense to do that as part of a separate PR with only those changes.

Reply:
They come with import of `github.com/linuxkit/linuxkit/src/cmd/linuxkit` (as we use `github.com/linuxkit/linuxkit/src/cmd/linuxkit/mobi` to extract data from label).